confdgnmi/src/confd_gnmi_adapter.py

```diff
@@ -1,7 +1,7 @@
 from abc import ABC, abstractmethod
 from dataclasses import dataclass
 from enum import Enum
-from threading import Event
+from queue import Queue
 
 from confd_gnmi_common import *
```

Comment:
`import *` is somewhat troublesome, is that necessary?

Reply:
Will be removed.